index.html

```diff
@@ -2612,6 +2617,96 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code title="">category</code> member
+        </h3>
+        <p>
+          The <dfn id="member-category"><code>category</code>
+          member</dfn> describes the expected application category to which the 
+          web application belongs. This is a hint to catalogs or stores of web 
```

Comment:
Good suggestion @kenchris.  I'll clarify that it can be a single category or multiple categories.